vllm/model_executor/layers/fused_moe/layer.py

```diff
@@ -353,6 +355,8 @@ def weight_loader(self, param: torch.nn.Parameter,
 
         # Case input scale: input_scale loading is only supported for fp8
         if "input_scale" in weight_name:
+            loaded_weight = loaded_weight.to(param.data.device)
```

Comment:
Can you leave a note here that this is needed for compressed-tensors only since we also support input scales for autoFP8